manager/integration/tests/test_basic.py

```diff
@@ -2842,11 +2989,11 @@ def test_backup_lock_restoration_during_deletion():  # NOQA
     5. Wait for the backup to be completed.
     6. Write more data to the volume and take another backup.
     7. Wait for the 2nd backup to be completed.
-    6. Delete the 1st backup.
-    7. Without waiting for the backup deletion completion, restore the 2nd
+    8. Delete the 1st backup.
+    9. Without waiting for the backup deletion completion, restore the 2nd
        backup from the backup store.
-    8. Verify the restored volume become faulted.
-    9. Wait for the 1st backup deletion and assert the count of the backups
+    10. Verify the restored volume become faulted.
```

Comment:
Not sure if this should become faulted, since the restore should be retried as long as the fromBackup flag is still set in the volume. not sure if and when we clean this up.

Reply:
Restore should be retried if the ```fromBackup``` flag has a non-existent backup. In this test, backup b2 deletion is in progress while restore is being initiated from backup b1 resulting in the restoring volume becoming faulted. 

However, to repro this scenario, we need a backup of at least 1.5 Gi of data and this test takes around 22 mins to run. Do we want to include this test or should we skip this?
@yasker What's your opinion on this?

Reply:
I would say skip the test for scheduled testing. But I would also have assumed that the restore of B1 on the new volume should be retried periodically :)

Reply:
We can skip this part and add them to the manual test instead.

Reply:
Skipped the test.